changelogs/fragments/10201-mssql_script-script-path-and-bom-removal.yml

```
@@ -0,0 +1,3 @@
+minor_changes:
+  - mssql_script - add ``script_path`` parameter to allow executing SQL scripts from files as an alternative to inline scripts.
+  - mssql_script - add automatic Byte Order Mark (BOM) detection and removal for script files to prevent SQL parsing errors.
```

The second fragment isn't needed, since this is part of the feature mentioned in the first fragment. Also the URLs are missing.
```suggestion
  - mssql_script - add ``script_path`` parameter to allow executing SQL scripts from files as an alternative to inline scripts (https://github.com/ansible-collections/community.general/issues/10201, https://github.com/ansible-collections/community.general/pull/10203).
```